<ui version="4.0" >
 <class>MouseWindow</class>
 <widget class="QMainWindow" name="MouseWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>524</width>
    <height>444</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>524</width>
    <height>444</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Fare Ayarları - Anatolya Bey</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../icons/startericons.qrc" >
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <property name="styleSheet" >
   <string notr="true" >QMainWindow {background-image: url(:/beyimage/beyimage.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>444</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>60</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title" >
     <string>Fare seçenekleri</string>
    </property>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../icons/startericons.qrc" >:/mouse-left/mouse-left.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>170</x>
       <y>20</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../../icons/startericons.qrc" >:/mouse-right/mouse-right.png</pixmap>
     </property>
    </widget>
    <widget class="QRadioButton" name="leftMouseSelection" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>180</y>
       <width>92</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Sol el</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rightMouseSelection" >
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>180</y>
       <width>92</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Sağ el</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="backButton" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Geri</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/go-previous/go-previous.png</normaloff>:/go-previous/go-previous.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton" >
    <property name="geometry" >
     <rect>
      <x>430</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>İptal</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/exit-button/application-exit.png</normaloff>:/exit-button/application-exit.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton" >
    <property name="geometry" >
     <rect>
      <x>320</x>
      <y>410</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>İleri</string>
    </property>
    <property name="icon" >
     <iconset resource="../../icons/startericons.qrc" >
      <normaloff>:/go-next/go-next.png</normaloff>:/go-next/go-next.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="clickSelectionGroup" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>280</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title" >
     <string>Tıklama seçenkleri</string>
    </property>
    <widget class="QRadioButton" name="singleClickSelection" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>20</y>
       <width>92</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Tek tıklama</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="doubleClickSelection" >
     <property name="geometry" >
      <rect>
       <x>20</x>
       <y>60</y>
       <width>92</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text" >
      <string>Çift tıklama</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>150</x>
       <y>10</y>
       <width>161</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text" >
      <string>Dosyaları çift tıklama (ilk tıklamada
dosyalar seçilir)
ilemi yoksa tek tıklama
ile mi açmak istiyorsunuz ?
-çift tıklama önerilir-</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="basementsLabel" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>110</y>
      <width>81</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Başlangıç&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Fare ayarları&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Panel ayarları&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Arkaplan&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Ağ ayarları&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Son&lt;/p>&lt;/body>&lt;/html></string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../icons/startericons.qrc" />
 </resources>
 <connections/>
</ui>
